Sources/AppcuesKit/Presentation/Public/ElementTargeting/AppcuesViewElement.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2023 Appcues. All rights reserved.
 //
 
-import Foundation
+import UIKit
```

Comment:
Is this just a convenience to get CoreGraphics and Foundation? I don't think UIKit actually used here, correct?

Reply:
Correct, I suppose this could be `import CoreGraphics`.